docker/proof-server-chart/values.yaml

```diff
@@ -0,0 +1,27 @@
+deployment:
```

Comment:
This globally looks great, but we may want to document the non-obivous / magic values in this file.